includes/blocks/class-sensei-course-outline-block.php

```diff
@@ -229,12 +236,12 @@ protected function render_lesson_block( $block ) {
 		$css = Sensei_Block_Helpers::build_styles( $block );
 
 		return '
-			<div ' . Sensei_Block_Helpers::render_style_attributes( $classes, $css ) . '>
-				<a
-				href="' . esc_url( get_permalink( $lesson_id ) ) . '">
+			<a href="' . esc_url( get_permalink( $lesson_id ) ) . '" ' . Sensei_Block_Helpers::render_style_attributes( $classes, $css ) . '>
+				<span>
 					' . $block['title'] . '
```

Comment:
Should this also be escaped?

Reply:
Looks like it was missing in a few places, I added a dedicated PR: #3681